### PR TITLE
Build and release aarch64

### PR DESCRIPTION
**Pull request by yitsushi**
Building `aarch64` is long with this approach. 30 minutes, installing
deps (libs, rust) is around 4-5 mins only, the rest spent on cargo
build.

### CODE REVIEW

**Comment by yitsushi** [comment]
Build on my fork:
https://github.com/yitsushi/firecracker/runs/5025564700?check_suite_focus=true

image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+name: build binaries (test)
```
Comment (Callisto13) [inline .github/workflows/build.yml]:
```suggestion
name: build binaries
```
guessing

Comment (Callisto13) [reply .github/workflows/build.yml]: oooh wait i see this is a separate job for testing that the build works on a workflow dispatch. nvm

**Review by Callisto13 — APPROVED** [review]
lgtm.

the `(test)` job separate from the release one is just there for when we want to trigger a manual thing to make sure it all works yes?

it might be nice to have some bash scripts so we don't have to change things in two places in the future

File: .github/workflows/build.yml (hunk 1 above)
Comment (yitsushi) [reply .github/workflows/build.yml]: Yes I left it there for debugging.

**Pull request merged**